mri/numerics/reconstruct.py

```diff
@@ -348,19 +348,12 @@ def sparse_rec_condatvu(gradient_op, linear_op, prox_dual_op, cost_op,
 
     # Get the final solution
     x_final = opt.x_final
-    if hasattr(linear_op, "transform"):
-        linear_op.transform.analysis_data = unflatten(
-            opt.y_final, linear_op.coeffs_shape)
-        transform_output = linear_op.transform
-    else:
-        linear_op.coeff = opt.y_final
-        transform_output = linear_op.coeff
     if hasattr(cost_op, "cost"):
         costs = cost_op._cost_list
     else:
         costs = None
 
-    return x_final, transform_output, costs, opt.metrics
```

Comment:
@LElgueddari , I have removed these as me and @zaccharieramzi feel that there is no use of this. The transform_Output is actually available in linear_op which is an input to this function.
Please correct me if there is any other future purpose of this

Reply:
I don't see any objection to this.

Reply:
I feel like you should return also `y_final` since it is the dual solution and that primal-dual method returns both the primal and the dual. Specifically for Condat Vu method.